src/main/java/com/mercadopago/client/point/PointClient.java

```diff
@@ -0,0 +1,92 @@
+package com.mercadopago.client.point;
+
+import static com.mercadopago.MercadoPagoConfig.getStreamHandler;
+import static com.mercadopago.serialization.Serializer.deserializeFromJson;
+
+import com.mercadopago.MercadoPagoConfig;
+import com.mercadopago.client.MercadoPagoClient;
+import com.mercadopago.core.MPRequestOptions;
+import com.mercadopago.exceptions.MPApiException;
+import com.mercadopago.exceptions.MPException;
+import com.mercadopago.net.HttpMethod;
+import com.mercadopago.net.MPHttpClient;
+import com.mercadopago.net.MPRequest;
+import com.mercadopago.net.MPResponse;
+import com.mercadopago.resources.point.PointPaymentIntent;
+import com.mercadopago.serialization.Serializer;
+import java.util.logging.Logger;
+import java.util.logging.StreamHandler;
+
+/** Client that use the Point APIs. */
+public class PointClient extends MercadoPagoClient {
+  private static final Logger LOGGER = Logger.getLogger(PointClient.class.getName());
+
+  private static final String PAYMENT_INTENT_URL =
+      "/point/integration-api/devices/%s/payment-intents";
+
+  /** Default constructor. Uses the default http client used by the SDK. */
+  public PointClient() {
+    this(MercadoPagoConfig.getHttpClient());
+  }
+
+  /**
+   * Constructor used for providing a custom http client.
+   *
+   * @param httpClient httpClient
+   */
+  public PointClient(MPHttpClient httpClient) {
+    super(httpClient);
+    StreamHandler streamHandler = getStreamHandler();
+    streamHandler.setLevel(MercadoPagoConfig.getLoggingLevel());
+    LOGGER.addHandler(streamHandler);
+    LOGGER.setLevel(MercadoPagoConfig.getLoggingLevel());
+  }
+
+  /**
+   * Method responsible for creating a payment intent.
+   *
+   * @param deviceId device id
+   * @param request attributes used to create a payment intent
+   * @return payment intent information
+   * @throws MPException an error if the request fails
+   * @see <a
+   *     href="https://www.mercadopago.com/developers/pt/reference/integrations_api_paymentintent_mlb/_point_integration-api_devices_deviceid_payment-intents/post">api
```

Comment:
o link muda dependendo do país, não só o domínio, o endpoint também
talvez seja legal deixar algo como
```suggestion
   *     href="https://www.mercadopago.com/developers/es/reference" > Mercado Pago Point
```

Reply:
Se o link muda dependendo do pais, talvez não faça sentido incluir a tag `@see` na documentação dos métodos com o link para a referência da api. Além de que esse link para a referência da API já está no readme.
Vi também que não são todos os clients do sdk que usam essa tag, então vou tirar a tag dos endpoints de Point e depois podemos criar uma tarefa para revisar essa documentação técnica, o que acha?

cc @romerosilva @eltinMeli

Reply:
Acho interessante o link do recurso estar diretamente no comentário, o que podemos adotar é uma linguagem padrão para nossa documentação, minha sugestão o inglês.

https://www.mercadopago.com/developers/en/reference/integrations_api_paymentintent_mlb/_point_integration-api_devices_deviceid_payment-intents/post

Reply:
o que muda não é só o idioma, da uma olhada

na referência de .com.ar
https://www.mercadopago.com.ar/developers/en/reference/integrations_api/_point_integration-api_devices_deviceid_payment-intents/post

na referência de .com.br
https://www.mercadopago.com.br/developers/en/reference/integrations_api_paymentintent_mlb/_point_integration-api_devices_deviceid_payment-intents/post